api/coda.py

```diff
@@ -564,6 +557,5 @@ def get_least_recently_asked_on_discord(
     questions: pd.DataFrame,
 ) -> pd.DataFrame:
     """Get all questions with oldest date and shuffle them"""
-    # pylint:disable=unused-variable
-    oldest_date = questions["last_asked_on_discord"].min()
-    return questions.query("last_asked_on_discord == @oldest_date")
+    _oldest_date = questions["last_asked_on_discord"].min()
+    return questions.query("last_asked_on_discord == @_oldest_date")
```

Comment:
(philosophical 🤔) probably too late now since it's used all over the place,, but I would consider magical macros* syntax as very nonpythonic when Python has a perfectly fine syntax for string template literals `f"last_asked_on_discord == {oldest_date}"` which the tools support without magic underscores for  ignoring stuff in static analysis / highlight / go-to-definition / ...

\* I am calling it macros because all arguments for/against C++ macros apply here IMHO, e.g. https://stackoverflow.com/a/14041847/1176601 .. and magic is anything against the second rule of zen of python for me

Reply:
Not really all over the place, definitely less than 10 places.
Yeah, it seems to work except you need to wrap the interpolated variable in another pair of quotes (`f"last_asked_on_discord == '{oldest_date}'"`).
Thanks, I will fix that